### PR TITLE
Fixed minor bugs in crypto rule behavior, assigngroups behavior, and added group review player_absent option

### PR DESCRIPTION
**Pull request by bleonar5**
Fixed minor bugs in crypto rule behavior, assigngroups behavior, and added group review player_absent option

### CODE REVIEW

File: public/js/cryptography.js [hunk 1]
```diff
@@ -50,93 +51,110 @@ var Cryptography = class Cryptography {
...
+          if(sym_count['A']+sym_count['B']+sym_count['C']+sym_count['D']+sym_count['E']+sym_count['F']+sym_count['G']+sym_count['H']+sym_count['I']+sym_count['J'] > 4 && round == 1){
+            console.log(rule+' broken');
+            rule_broken += 1;}
           break;
         case 2:
           console.log(sym_count['A']+sym_count['B']+sym_count['C']+sym_count['D']+sym_count['E']+sym_count['F']+sym_count['G']+sym_count['H']+sym_count['I']+sym_count['J'])
           console.log(round);
-          if(sym_count['A']+sym_count['B']+sym_count['C']+sym_count['D']+sym_count['E']+sym_count['F']+sym_count['G']+sym_count['H']+sym_count['I']+sym_count['J'] < 3 && round == 1)
-            rule_broken = rule;
+          if(sym_count['A']+sym_count['B']+sym_count['C']+sym_count['D']+sym_count['E']+sym_count['F']+sym_count['G']+sym_count['H']+sym_count['I']+sym_count['J'] < 3 && round == 1){
```
Comment (joshbenhamou) [inline public/js/cryptography.js:84]: Not saying we do it now but we could definitely improve this code in the future

File: resources/views/layouts/participants/tasks/cryptography-group.blade.php [hunk 2]
```diff
@@ -296,9 +296,12 @@ function clear_timer(){
     //IF USER IN THIS GROUP
     if (data['user']['group_id'].toString() == '{{ $user->group_id }}'){
       //INFORM
+      $('#rule_break_num').text(data['rule_broken'].toString());
+      $('#penalty_amt').text((2*data['rule_broken']).toString());
```
Comment (joshbenhamou) [inline resources/views/layouts/participants/tasks/cryptography-group.blade.php:300]: what does amt stand for here?

**Review by joshbenhamou — APPROVED** [review]
Mostly using these MRs to understand the codebase more. There's a lot of optimization we could do but I suggest we punt on it while we are in the final stretch of this study

File: resources/views/layouts/participants/tasks/cryptography-group.blade.php (hunk 2 above)
Comment (bleonar5) [reply resources/views/layouts/participants/tasks/cryptography-group.blade.php:300]: amount

**Pull request merged**